filament/src/View.cpp

```diff
@@ -574,10 +574,6 @@ void FView::prepare(FEngine& engine, backend::DriverApi& driver, ArenaScope& are
     s.fogInscatteringSize  = fogOptions.inScatteringSize;
     s.fogColorFromIbl      = fogOptions.fogColorFromIbl ? 1.0f : 0.0f;
 
-    // upload the renderables's dirty UBOs
-    engine.getRenderableManager().prepare(driver,
-            renderableData.data<FScene::RENDERABLE_INSTANCE>(), merged);
-
```

Comment:
Interesting that this can be removed.  I don't totally get it, but I believe it.  :)

Reply:
Yeah, this is because before we were storing the data in a staging "heap" area, but now we issue a driver command each time the user calls setBones.